src/presign/participant.rs

```diff
@@ -926,10 +926,10 @@ impl PresignKeyShareAndInfo {
         };
 
         let r1_private = round_one::Private {
-            k,
-            rho,
+            k: k.into(),
```

Comment:
I don't think we should have `From` implementation do this. It hides what the conversion is about here: BigNumber -> SecretBigNumber. We should use a constructor to create the SecretBigNumber explicitly. E.g. `SecretBigNumber::from_number`.